interface/modules/custom_modules/oe-module-faxsms/src/Events/NotificationEventListener.php

```diff
@@ -0,0 +1,91 @@
+<?php
+
+/**
+ * Fax SMS Module Member
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Jerry Padgett <sjpadgett@gmail.com>
+ * @copyright Copyright (c) 2023 Jerry Padgett <sjpadgett@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General public License 3
+ */
+
+namespace OpenEMR\Modules\FaxSMS\Events;
+
+use MyMailer;
+use OpenEMR\Events\Messaging\SendNotificationEvent;
+use OpenEMR\Modules\FaxSMS\Controller\AppDispatch;
+use OpenEMR\Modules\FaxSMS\Controller\TwilioSMSClient;
+use OpenEMR\Services\DocumentTemplates\DocumentTemplateService;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class NotificationEventListener implements EventSubscriberInterface
+{
+    private null|TwilioSMSClient $clientApp;
+    private DocumentTemplateService $docClient;
+
+    public function __construct()
+    {
+        $this->clientApp = AppDispatch::getApiService('sms');
+        $this->docClient = new DocumentTemplateService();
+    }
+
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            SendNotificationEvent::class => 'onNotifyEvent',
+        ];
+    }
+
+    public function subscribeToEvents(EventDispatcher $eventDispatcher)
+    {
+        $eventDispatcher->addListener('sendNotification.sendSMS', [$this, 'onNotifyEvent']);
```

Comment:
@sjpadgett So it looks like I would just need to fire off the sendNotifiction.sendSMS event when the appointment is saved.  

I'm wondering if it makes sense to name this sendNotification.send since it handles both SMS and email?

Reply:
renamed. will need to pass in args (see example in event class) to support what is included in message  minimum is pid and a message body.